controllers/openshift_controller.go

```diff
@@ -1527,8 +1533,8 @@ func (r *KataConfigOpenShiftReconciler) SetupWithManager(mgr ctrl.Manager) error
 			&McpEventHandler{r}).
 		Watches(
 			&source.Kind{Type: &corev1.Node{}},
-			&NodeEventHandler{r}).
-		Complete(r)
+			&NodeEventHandler{r},
+		).Complete(r)
```

Comment:
What if user switches a feature at some point? should we support it? if we do we should watch ConfigMaps here

Reply:
Right. Makes sense and I just sent.